metricbeat/metricbeat.reference.yml

```diff
@@ -492,11 +492,18 @@ metricbeat.modules:
 #------------------------------- Windows Module ------------------------------
 - module: windows
   metricsets: ["perfmon"]
+  enabled: true
   period: 10s
+  perfmon.ignore_non_existent_counters: true
   perfmon.counters:
+  #  - instance_label: processor.name
+  #    instance_name: total
+  #    measurement_label: processor.time.total.pct
+  #    query: '\Processor Information(_Total)\% Processor Time'
 
 - module: windows
   metricsets: ["service"]
+  enabled: true
   period: 60s
```

Comment:
Now it becomes interesting. In the `reference` config we should use defaults and only in `config.yml`overwrite the defaults. At least that is what we did so far.

Reply:
This is how periods were in the reference config before this PR :)

I think that supporting different default periods can make a lot of sense (this module is a good example), but this gives for a bigger discussion out of the scope of this PR, wdyt?

Reply:
That we had different periods in the reference config was more an accident as the values were taken from the short config. Now that we have a reference file we could use the default one.

There are a lot of interesting discussion popping up with our changes here. We can take these offline. Will merge as is for now.